addons/sale_project/views/sale_order_views.xml

```diff
@@ -22,7 +22,7 @@
                 <button type="object" name="action_view_project_ids" class="oe_stat_button" icon="fa-puzzle-piece" invisible="not show_project_button" groups="project.group_project_user">
                     <field name="project_count" widget="statinfo" string="Projects"/>
                 </button>
-                <button class="oe_stat_button" name="action_view_milestone" type="object" icon="fa-check-square-o"  invisible="not is_product_milestone or not project_ids" groups="project.group_project_milestone">
+                <button class="oe_stat_button" name="action_view_milestone" type="object" icon="fa-check-square-o" invisible="not is_product_milestone or not project_ids or state == draft" groups="project.group_project_milestone">
```

Comment:
Are you sure it works as you did? 🤔 

I think `draft` should be a string, no?

```suggestion
                <button class="oe_stat_button" name="action_view_milestone" type="object" icon="fa-check-square-o" invisible="not is_product_milestone or not project_ids or state == 'draft'" groups="project.group_project_milestone">
```

Reply:
hello @xavierbol,
changes are done,
Thanks